_includes/master/reqs-sys.md

```diff
@@ -56,3 +55,20 @@ Ensure that your hosts and firewalls allow the necessary traffic based on your c
 
 \* _If your compute hosts connect directly and don't use IP-in-IP, you don't need to allow IP-in-IP traffic._
 {% endif -%}
+
+## Privileges
+
+Ensure that {{site.prodname}} is provided the following privileges.
+
+- `CAP_SYS_ADMIN`
```

Comment:
Since this list contains only one item, it is, by definition, not a list. It would be more graceful to fold this into the sentence above, for example:
`Ensure that {{site.prodname}} is provided the `CAP_SYS_ADMIN` privilege.`

And on line 65, it would be a singular "privilege".

Reply:
I would also suggest to avoid the passive voice here in a construction such as s/is provided/has